.ado/jobs/desktop.yml

```diff
@@ -19,6 +19,14 @@ jobs:
           (FullyQualifiedName!=RNTesterIntegrationTests::WebSocket)&
           (FullyQualifiedName!~WebSocketModule_)&
           (FullyQualifiedName!=WebSocketResourcePerformanceTest::ProcessThreadsPerResource)
+      # HostFunctionTest              - Crashes under JSI/V8
```

Comment:
Could you file an issue about these failing?

Reply:
Will do.